doc/_static/version_switch.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 PSF. Licensed under the PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+// File originates from the cpython source found in Doc/tools/sphinxext/static/version_switch.js
+
+(function() {
+  'use strict';
+
+  //var doc_url = "www.pygmt.org";
+  var doc_url = "0.0.0.0:8000"; // for local testing only
+  var url_re = new RegExp(doc_url + "\\/(dev|latest|(v\\d+\\.\\d+\\.\\d+))\\/");
+  // List all versions.
+  // Add one entry "version: title" for any minor releases
+  var all_versions = {
+    'latest': 'latest',
+    'dev': 'dev',
+    'v0.2.0': 'v0.2.0',
+    'v0.1.2': 'v0.1.2',
+    'v0.1.1': 'v0.1.1',
+    'v0.1.0': 'v0.1.0',
+  };
```

Comment:
The main reason is that the version is `0.0.1a0` without the leading `v`. So it's different from the other versions and the regex doesn't work for it. 

In commit 882394fd367a9719784f77b304342a282cb98a01, I added an entry for 0.0.1a0 so that it will be shown in the dropdown. Now you can find 0.0.1a0 in the dropdown and can switch to 0.0.1a0, but can't switch back.